contracts/BSM/Investors/AMM/AMMInvestorsHelper.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
```

Comment:
what about tests? I feel like just reading the code here is not good enough, if we will merge it to master it's considered tested and verified.
I'm afraid to do it this way and I think it will better to merge this code as part of the zap implementation, including all the relevant tests

Reply:
Let's take it offline